src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+
+  &--1 :nth-child(-n+1),
+  &--2 :nth-child(-n+2),
+  &--3 :nth-child(-n+3),
+  &--4 :nth-child(-n+4),
+  &--5 :nth-child(-n+5) {
+    background-image: url(/images/star-active.svg);
+  }
```

Comment:
Use `@for in `loop, instead

Reply:
Agree, use [loop](https://sass-lang.com/documentation/at-rules/control/for)